packages/next/server/incremental-cache.ts

```diff
@@ -82,7 +83,8 @@ export class IncrementalCache {
       this.cache = new LRUCache({
         max,
         length({ value }) {
-          if (!value || value.kind === 'REDIRECT') return 25
+          if (!value || value.kind === 'REDIRECT' || value.kind === 'IMAGE')
+            return 25
```

Comment:
25 seems pretty low. Should we use `buffer.length` kind of like the PAGE kind is using `html.length`?

Reply:
Updated to throw an invariant here since the image kind should never end up here